python/caffe/_caffe.cpp

```diff
@@ -15,6 +15,7 @@
 #include <fstream>  // NOLINT
 
 #include "caffe/caffe.hpp"
+#include "caffe/common.hpp"
```

Comment:
This is included by `caffe.hpp`... I'd probably omit it here (unless lint really wants it).